testpmd/hooks/roles/mirror-rh-nfv/tasks/main.yml

```diff
@@ -60,4 +60,17 @@
     - update_status == ['True']
   retries: 60
   delay: 60
+  ignore_errors: true
+
+- name: Retrieve more logs in the case of failure
```

Comment:
Isn't it better to save the output in a file and upload it to DCI?

Reply:
yeah, maybe just create a file with all the information. What about adding this something like this for each worker?
To verify with nodes is not rendering the policy.
```
$ oc describe node worker-X | grep Config
                    machineconfiguration.openshift.io/currentConfig: rendered-worker-1dc01cb5064cb58b21917f42cdb00f5e
                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-1dc01cb5064cb58b21917f42cdb00f5e
```
Thanks

Reply:
checked that on cluster6
```
$ for i in {0..3}; do echo "worker-$i" && oc describe node worker-$i | grep Config; done
worker-0
                    machineconfiguration.openshift.io/currentConfig: rendered-worker-f37dd7c665eef52f2c38899c17bdbc8c
                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-f37dd7c665eef52f2c38899c17bdbc8c
worker-1
                    machineconfiguration.openshift.io/currentConfig: rendered-worker-bcecdf2630eefa01ecc405afd90efad7
                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-f37dd7c665eef52f2c38899c17bdbc8c
worker-2
                    machineconfiguration.openshift.io/currentConfig: rendered-worker-bcecdf2630eefa01ecc405afd90efad7
                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-f37dd7c665eef52f2c38899c17bdbc8c
worker-3
                    machineconfiguration.openshift.io/currentConfig: rendered-worker-bcecdf2630eefa01ecc405afd90efad7
                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-f37dd7c665eef52f2c38899c17bdbc8c
```